openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -391,12 +391,13 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #88.                    if 'out_nd' in d_outputs:
</pre><p>If statment is repeated twice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>fixed</p>

Reply:
fixed